### ISSUE TITLE
IPagedList not extending IEnumerable

### ISSUE THREAD

**Issue by Jecoms** [kpi-ua/X.PagedList#49]
IEnumerable extensions no longer work using current version.

**Comment by Mardoxx** [comment]
Can you provide more detail?

I would assume you mean it is kicking off in your `Views` saying something along the lines of `The type 'IEnumerable<>' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0`?

**Comment by Jecoms** [comment]
Yes. The exposed namespace is now also X.PagedList in the current version. None of the IEnumerable extensions are available in version 5.x. Is the only current solution to patch my .config files or is the current version targeting a different version of .NET MVC? I have no problem sticking with the <5 PagedList and PagedList.Mvc for the time being.

**Comment by Mardoxx** [comment]
I had the same issue upgrading from an old version. I believe this has something to do with including a PCL in your project. Not entirely sure. To fix it I updated `X.PagedList`, closed Visual Studio, deleted (renamed) the hidden `.vs`  folder in the soluton directory, then opened the project and it compiled fine.

The namespace _has_ changed to `X.PagedList` somewhere down the line (you can always do a case sensitive find and replace `using PagedList` for `using X.PagedList`). The info page on GitHub hasn't been updated yet. The demo project uses `X.PagedList`

**Comment by Jecoms** [comment]
Well, just like the error suggested, I only needed to add the assembly reference to the compilation XML tag in the project web.config file. I still would like to know why this patch is needed. Is it something that VS will handle in the future or is it an issue with how the PCL was configured/targeted?

This is all I added (besides updating refs to X.PagedList) and it builds/runs fine now. This should be in a FAQ or the readme.

`<assemblies>`
 `<add assembly="System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />`
`</assemblies>`
